docs/en/alab/test_project_configuration.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: docs
+comment: no
+header: true
+seotitle: Annotation Lab | John Snow Labs
+title: Test Project Configuration
+permalink: /docs/en/alab/test_project_configuration
+key: docs-training
+modify_date: "2022-11-20"
+use_language_switcher: "Python-Scala"
+show_nav: true
+sidebar:
+  nav: annotation-lab
+---
+
+A **Project Owner** or a **Manager** can use the completed tasks (completions) from a project to test the pre-trained model. The project "Test Configuration" feature can be found on the train page, accessible from the Project Menu. During the training, a progress bar is shown on the top of the train page to show the status of the 
```

Comment:
"During the training, a progress bar is shown on the top of the train page to show the status of the" it seems this sentence is incomplete. Also isn't it suppose to be testing rather than training?